src/expression-helpers.js

```diff
@@ -27,7 +27,7 @@ function computeHasDependencies(compute){
 function getObservableValue_fromDynamicKey_fromObservable(key, root, helperOptions, readOptions) {
 	// This needs to return something similar to a ScopeKeyData with intialValue and parentHasKey
 	var getKeys = function(){
-		return stacheKey.reads(("" + canReflect.getValue(key)).replace(".", "\\."));
+		return stacheKey.reads(("" + canReflect.getValue(key)).replace(/\./g, "\\."));
```

Comment:
could we just return [{key: canReflect.getValue(key) }]

```suggestion
		return [{key: ""+canReflect.getValue(key)}]
```

Reply:
No, this would break functions. `stacheKey.reads("@toFixed")` -> `{ key: "toFixed", at: true }`.